packages/node/src/cron/node-cron.ts

```diff
@@ -30,20 +30,23 @@ export interface NodeCron {
  */
 export function instrumentNodeCron<T>(lib: Partial<NodeCron> & T): T {
   return new Proxy(lib, {
-    get(target, prop: keyof NodeCron) {
+    get(target, prop) {
       if (prop === 'schedule' && target.schedule) {
         // When 'get' is called for schedule, return a proxied version of the schedule function
         return new Proxy(target.schedule, {
           apply(target, thisArg, argArray: Parameters<NodeCron['schedule']>) {
             const [expression, callback, options] = argArray;
 
-            if (!options?.name) {
+            const name = options?.name;
+            const timezone = options?.timezone;
+
+            if (!name) {
               throw new Error('Missing "name" for scheduled job. A name is required for Sentry check-in monitoring.');
             }
 
-            async function monitoredCallback(): Promise<void> {
+            const monitoredCallback = async (): Promise<void> => {
```

Comment:
@AbhiPrasad (I think you worked on this initially) is it important that this is not an arrow function? Without this, it complains about stuff being mutateable as this is hoisted, I guess.

Reply:
arrow function should be fine, it shouldn't rely on the `this` context.